src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment/HoloLensTrackingObserver.cs

```diff
@@ -15,15 +15,15 @@ public override TrackingState TrackingState
         {
             get
             {
-#if UNITY_WSA
+#if UNITY_EDITOR
```

Comment:
>UNITY_EDITOR [](start = 4, length = 12)

Was this mainly a change to get alphabetical ordering for the ifdefs? Functionally it seems the same (AFAICT)

Reply:
it feels like this may be a product of a weird merge. I do not know why this changed.